AniHyou/Screens/MediaList/ViewModels/MediaListViewModel.swift

```diff
@@ -27,27 +29,32 @@ class MediaListViewModel: ObservableObject {
     @Published var isLoading = false
     
     var filteredMediaList: [UserMediaListQuery.Data.Page.MediaList?] {
-        if searchText.isEmpty {
-            return mediaList
-        } else {
+        if searchText.count > 0 && searchText.count < 3 {
+            return Array()
+        } else if searchText.count >= 3 {
             let filtered = mediaList.filter {
                 let title = $0?.media?.title?.userPreferred
                 if title == nil || title?.isEmpty == true {
                     return false
                 }
                 return title!.lowercased().contains(searchText.lowercased())
             }
-            if hasNextPage && filtered.count < 25 {
```

Comment:
don't remove the `hasNextPage` check

Reply:
yeah just playing with something here...